GHA: buildDocker (self-hosted, ARM64) step failing

Last success was 2 weeks ago: https://github.com/hyperledger/besu/actions/runs/8487812388

Discovered during the 24.3.2 release process -
https://github.com/hyperledger/besu/actions/runs/8639188327/job/23686748688#step:9:478

It actually failed for 24.4.0-RC2 as well last week: https://github.com/hyperledger/besu/actions/runs/8500195660

```

INFO: Found goss_wait.yaml, waiting for it to pass before running tests
ERROR: goss_wait.yaml never passed
INFO: Deleting container

> Task :testDocker FAILED

FAILURE: Build failed with an exception.

* Where:
Build file '/home/runner/_work/besu/besu/build.gradle' line: 798

* What went wrong:
Execution failed for task ':testDocker'.
> Process 'command '/bin/bash'' finished with non-zero exit value 1
```

--- a/docker/test.sh
+++ b/docker/test.sh
@@ -20,7 +20,7 @@ if [[ $i != 0 ]]; then exit $i; fi
 # we test that things listen on the right interface/port, not what interface the advertise
 # hence we dont set p2p-host=0.0.0.0 because this sets what its advertising to devp2p; the important piece is that it defaults to listening on all interfaces
 GOSS_FILES_PATH=$TEST_PATH/01 \
-bash $TEST_PATH/dgoss run $DOCKER_IMAGE \
+bash $TEST_PATH/dgoss run -p 8545:8545 -p 8546:8546 -p 8547:8547 -p 30303:30303 $DOCKER_IMAGE \
 --network=dev \
 --rpc-http-enabled \
 --rpc-ws-enabled \

--- a/docker/test.sh
+++ b/docker/test.sh
@@ -20,7 +20,7 @@ if [[ $i != 0 ]]; then exit $i; fi
 # we test that things listen on the right interface/port, not what interface the advertise
 # hence we dont set p2p-host=0.0.0.0 because this sets what its advertising to devp2p; the important piece is that it defaults to listening on all interfaces
 GOSS_FILES_PATH=$TEST_PATH/01 \
-bash $TEST_PATH/dgoss run -p 8545:8545 -p 8546:8546 -p 8547:8547 -p 30303:30303 $DOCKER_IMAGE \
+bash $TEST_PATH/dgoss run --sysctl net.ipv6.conf.all.disable_ipv6=1 $DOCKER_IMAGE \
 --network=dev \
 --rpc-http-enabled \
 --rpc-ws-enabled \

